pkg/sfu/downtrack.go

```diff
@@ -836,25 +793,23 @@ func (d *DownTrack) UpdateStats(packetLen uint32) {
 	d.packetCount.add(1)
 }
 
-// bandwidthConstrainedMute enables or disables media forwarding dictated by channel bandwidth constraints
-func (d *DownTrack) bandwidthConstrainedMute(val bool) {
-	if d.bandwidthConstrainedMuted.get() == val {
-		return
-	}
-	d.bandwidthConstrainedMuted.set(val)
-	if val {
-		d.reSync.set(val)
-	}
-}
-
 func (d *DownTrack) writeSimpleRTP(extPkt *buffer.ExtPacket) error {
 	if d.reSync.get() {
 		if d.Kind() == webrtc.RTPCodecTypeVideo {
+			if d.TargetSpatialLayer() == InvalidSpatialLayer {
```

Comment:
Simple RTP tracks also get the same layer check treatment. Just that there is only one layer. Am hoping we can combine the methods in a subsequent PR.